backend/impl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -343,10 +344,23 @@ IGitMacheteRepositorySnapshot createSnapshot(IBranchLayout branchLayout) throws
           Case($(GitCoreRepositoryState.BISECTING), OngoingRepositoryOperation.BISECTING),
           Case($(), OngoingRepositoryOperation.NO_OPERATION));
 
+      val operationsBaseBranchName = deriveOngoingOperationsBaseBranchName(ongoingOperation);
+
       return new GitMacheteRepositorySnapshot(List.narrow(rootBranches), branchLayout, currentBranchIfManaged,
-          managedBranchByName, duplicatedBranchNames, skippedBranchNames, gitCoreRepository.deriveRebasedBranch(),
-          preRebaseHookExecutor,
-          ongoingOperation);
+          managedBranchByName, duplicatedBranchNames, skippedBranchNames, preRebaseHookExecutor,
+          ongoingOperation, operationsBaseBranchName);
+    }
+
+    @SneakyThrows
```

Comment:
Hmm I'd avoid `@SneakyThrows` in production (non-test) code unless there's a good reason too... what's the exception that can be thrown from this method's body?

Reply:
Generally I'd rather wrap the exception into `GitMacheteException`, as done in other methods